framework/common/src/main/java/edu/kit/kastel/mcse/ardoco/core/api/diagramrecognition/Classification.java

```diff
@@ -12,7 +12,7 @@ public enum Classification {
     LABEL("Label"), CLASS("ClassNode"),
 
     // Artificial Classes
-    TEXT("TEXT"), UNKNOWN(null);
+    TEXT("TEXT"), RAWTEXT("RAWTEXT"), UNKNOWN(null);
```

Comment:
I honestly have no clue. Probably something vestigial that i accidentally merged from some older version? Ill remove it

Reply:
Nevermind. I am completely confused. This is currently already part of [Classification](https://github.com/ArDoCo/Core/blob/main/framework/common/src/main/java/edu/kit/kastel/mcse/ardoco/core/api/diagramrecognition/Classification.java). I have no clue why this even shows as an addition, since it is part of current main branch already. It seems to be something you added about 3 months ago in commit 8630c2bf2ec1f207ce8100332696d82cdf710bb8

Reply:
Ok . Maybe the diff was wrong :)